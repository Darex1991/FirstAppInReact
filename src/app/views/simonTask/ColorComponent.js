import React, { Component, PropTypes } from 'react';
import { Form, Text, Checkbox } from 'react-form'

export default class ColorComponent extends Component {
  render() {
    return (
      <div className="profile default-input">
        <h2> Color Schema </h2>

        <div>
          <Checkbox field='darkColorSchema'/>
          <p> Dark schema </p>
        </div>
      </div>
    )
  }
}
