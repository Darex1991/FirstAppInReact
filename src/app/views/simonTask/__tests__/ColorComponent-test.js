import React from 'react';
import renderer from 'react-test-renderer';
import ColorComponent from '../ColorComponent';
test('Should render ColorComponent correctly', () => {
    const tree = renderer.create(
        <ColorComponent /> 
    ).toJSON();
    expect(tree).toMatchSnapshot();
});
    