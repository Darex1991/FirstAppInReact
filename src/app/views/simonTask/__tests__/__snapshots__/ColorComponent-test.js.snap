exports[`test Should render ColorComponent correctly 1`] = `
<div
  className="profile default-input">
  <h2>
    Color Schema
  </h2>
  <div>
    <FormInputCheckbox
      field="darkColorSchema" />
    <p>
      Dark schema
    </p>
  </div>
</div>
`;
